src/explorepy/explore.py

```diff
@@ -175,6 +183,9 @@ def stop_recording(self):
                 self.recorders['marker'].stop()
             if self.recorders['timer'].is_alive():
                 self.recorders['timer'].cancel()
+
+            self.recorders['meta'].write_meta()
```

Comment:
Maybe it would be better to move `write_meta` to the beginning of the recording to make sure the metadata is not lost in case of sudden program termination.

Reply:
Can do, I was going with the philosophy that if my recording breaks for any reason, then I don't want meta data about it. But can move it up

Reply:
@bmeatayi Can you confirm this is what you really want?

Reply:
If the recording breaks, I think users would prefer to have access to the recorded data anyway. Imagine you have recorded hundreds of trials in one session and suddenly the program freezes. You would still want to analyze the recorded trials anyway.